docs/Telegram.md

```diff
@@ -1,73 +1,68 @@
+
+### Speciality groups
+
+For those interested in certain equipment, specialities, or hobbies.
+
+| Group name                                                  | Purpose                                                                                                                                     |
+| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Woodwork](https://t.me/joinchat/AYtZgkk7n1MqvkN9N2fmsA)    | For any chatter relating to Woodworking, or matters relating to our Wood workshop.                                                          |
+| [Metalwork](https://t.me/joinchat/B8-OC1MTETWTM8vBexJOag)   | Similarly, anything relating to Metalwork or our Metal workshop.                                                                            |
+| [Electronics](https://t.me/+WLCJFgPE5LHX69XW)               | Anything relating to electronics in general, or our Electronics area.                                                                       |
+| [Visual Arts](https://t.me/hacmanvisualarts)                | Relating to our Visual Arts area,  any of the specialities within it, or visual arts in general.                                            |
+| [3D printing](https://t.me/joinchat/DZNJNRJimIP7XoyvDArnUg) | All things to do with 3D printing, or our own 3D printers.                                                                                  |
+| [Bike space](https://t.me/+hy1MtdfETqwzZmVk)                | For those with an interest in bikes, biking, or our collection of bike tools.                                                               |
+| [Games Night](https://t.me/+Q5e1aiVMwDvND71B)               | Board gaming & game night related chatter.                                                                                                  |
+| [Clutter Exchange](https://t.me/+11IEnLLuh8QxYWM0)          | See if any other members might be interest in anything you're wanting rid of: tools, equipment, material, or anything else possibly useful. |
+
+### Operations
+
+We have a number of groups specifically for the operation of the Hackspace overall.
+
+Some of these groups are private by default – in which case we'll list their existance, but you won't be able to join them from this page.
+
+| Group name                                                     | Purpose                                                                                                              |
+| -------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| [Laser Training](https://t.me/joinchat/D5l4WUNG_uyg6YB0PYikRA) | To coordinate Laser training sessions between trainers and those waiting to be inducted.                             |
```

Comment:
This is going against the previous system where people would request an induction on the membership system, and the tool page would then present this link to them. This ensured that a training record was created appropriately.